extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/TransactionManagerConfiguration.java

```diff
@@ -32,6 +33,20 @@ public final class TransactionManagerConfiguration {
     @ConfigItem(defaultValue = "ObjectStore")
     public String objectStoreDirectory;
 
+    /**
+     * The type of object store.
+     */
+    @ConfigItem(defaultValue = "FileSystem")
```

Comment:
FWIW Quarkus uses hyphenated strings for enums by default.

Also, I think you should use `object-store.type` here, to be consistent with `object-store.datasource`?

```suggestion
    @ConfigItem(name = "object-store.type", defaultValue = "file-system")
```